okhttp-logging-interceptor/src/test/java/okhttp3/logging/HttpLoggingInterceptorTest.java

```diff
@@ -641,6 +642,26 @@ private void bodyGetNoBody(int code) throws IOException {
         .assertNoMoreLogs();
   }
 
+    @Test public void connectFail() throws IOException {
+        setLevel(Level.BASIC);
+        url = new HttpUrl.Builder()
+            .scheme("http")
+            .host("okhttp-notfound.com")
```

Comment:
sadly when you run tests like this in certain environments, you get a result. I’ve seen it on Comcast. There are other special values that’ll work though . . . lemme look it up.